grafana/block_verification.json

```
@@ -25,10 +25,9 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": null,
+      "decimals": 6,
```

```suggestion
      "decimals": 0,
```

Forgot this one. Looks prettier without decimals since it's a block height :grin: